src/lib/lib.jl

```diff
@@ -1,5 +1,14 @@
 using Base: RefValue
 
+if VERSION > v"1.7.0-DEV.204"
+  using Base: ismutabletype
+else
+  function ismutabletype(@nospecialize(t::Type))
```

Comment:
`@pure` this?

Reply:
It's copied straight from Base, I see this was discussed here: https://github.com/JuliaLang/julia/pull/39037/files#r550261314